ResearchKit/Common/ORKNavigableOrderedTask.m

```diff
@@ -178,6 +178,9 @@ - (ORKStep *)stepAfterStep:(ORKStep *)step withResult:(ORKTaskResult *)result {
     if ([navigationRules count] > 0 && nextStep == nil) {
         ORKStepNavigationRule *rule = navigationRules.firstObject;
         NSString *stepID = [rule defaultStepIdentifier];
+        if ([stepID isEqualToString:@"-1"]) {
+            stepID = @"ThankYouScreen";
```

Comment:
Friendly note: if you use the ```-1``` in several places it will be good to have them stored at least as a constant :)

Reply:
gotcha ;)